Dockerfile

```diff
@@ -41,3 +37,52 @@ RUN bundle install --jobs 20 --retry 5
 
 # Copy the main application.
 COPY . ./
+
+# Remove extra files
+RUN rm -rf /usr/local/bundle/cache/*
+
+
+##############################
+# PACKAGE STAGE              #
+##############################
+FROM ruby:3.2-alpine
+LABEL maintainer=engineering@kodeco.com
+LABEL com.github.actions.name="robles"
+LABEL com.github.actions.author="Kodeco <engineering@kodeco.com>"
+LABEL com.github.actions.description="Content publication for kodeco.com"
+LABEL com.github.actions.color="purple"
+LABEL com.github.actions.icon="book"
+
+ARG APP_ROOT=/app/robles
+ARG RUBY_ENV=${RUBY_ENV:-production}
+
+ENV RUBY_ENV=${RUBY_ENV}
+
+ARG RUNTIME_PACKAGES="imagemagick git tzdata"
+ARG TEST_AND_DEV_PACKAGES="bash build-base libsodium-dev"
```

Comment:
Maybe we don't need `libsodium-dev` here as we are already copying `libsodium` from builder below.

Reply:
I wasn't sure whether it might also need header files or something else that might be part of the package when building a new version of the gem? Since this was only for test and development I thought it could probably stay in.